common/src/main/java/com/microsoft/identity/common/AndroidCommonComponents.java

```diff
@@ -147,4 +148,40 @@ public synchronized IDevicePopManager getDefaultDevicePopManager() throws Client
                 exception
         );
     }
+
+    @Override
+    public <T> INameValueStorage<T> getNameValueStore(String storeName, Class<T> clazz) {
+        return getEncryptedNameValueStore(storeName, null, clazz);
+    }
+
+    @Override
+    public <T> INameValueStorage<T> getEncryptedNameValueStore(String storeName, IKeyAccessor helper, Class<T> clazz) {
+        final IKeyBasedStorage mgr = SharedPreferencesFileManager.getSharedPreferences(mContext, storeName, helper);
+        if (Long.class.isAssignableFrom(clazz)) {
+            @SuppressWarnings("unchecked")
+            final INameValueStorage<T> store = (INameValueStorage<T>) new SharedPreferenceLongStorage(mgr);
+            return store;
+        } else if (String.class.isAssignableFrom(clazz)) {
+            @SuppressWarnings("unchecked")
+            final INameValueStorage<T> store = (INameValueStorage<T>) new SharedPrefStringNameValueStorage(mgr);
+            return store;
+        }
+        throw new UnsupportedOperationException("Only Long and String are natively supported as types");
+    }
+
+    @Override
+    public IKeyBasedStorage getEncryptedFileStore(String storeName, IKeyAccessor helper) {
+        return SharedPreferencesFileManager.getSharedPreferences(mContext, storeName, helper);
+    }
+
+    @Override
+    public IKeyBasedStorage getFileStore(String storeName) {
+        return SharedPreferencesFileManager.getSharedPreferences(mContext, storeName, null);
+    }
+
+    @Deprecated
+    @Override
+    public Context getPlatformContext() {
+        return mContext;
+    }
```

Comment:
This is being added in this PR, right? Why deprecated?

IIRC, you were going to just remove this method all together?

Reply:
It should be gone.